pages/docs/guides/create-retention-curves.mdx

```diff
@@ -175,7 +175,7 @@ A few notes on the resulting columns:
   width="100%"
   height="500px"
```

Comment:
This embedded table currently looks pretty bad–it's tiny and you have to scroll within the table. could you try changing the height and seeing if it helps? 

<img width="870" alt="Screenshot 2023-09-11 at 4 52 58 PM" src="https://github.com/hashboard-hq/documentation/assets/34147666/6daa0574-25c6-42d0-a5be-f8f53f5b98cf">

Reply:
going to try using public link instead